prog/weave-npc/main.go

```diff
@@ -179,7 +170,7 @@ func createBaseRules(ipt *iptables.IPTables, ips ipset.Interface) error {
 
 	ruleSpecs := [][]string{
 		{"-m", "state", "--state", "RELATED,ESTABLISHED", "-j", "ACCEPT"},
-		{"-m", "state", "--state", "NEW", "-m", "set", "!", "--match-set", npc.LocalIpset, "src", "-j", "RETURN"},
+		{"-m", "physdev", "--physdev-in=vethwe-bridge", "-j", "RETURN"},
```

Comment:
I'm not so sure this rule can move.  But again, hard-coding the device name is wrong.